tests/repro_tests/frontend/detector_descriptor/test_sift_reproducibility.py

```diff
@@ -0,0 +1,13 @@
+""""Reproducibility tests for SIFT descriptor.
+
+Authors: Ayush Baid
+"""
+import tests.repro_tests.frontend.detector_descriptor.test_detector_descriptor_reproducibility_base as test_detector_descriptor_reproducibility_base
```

Comment:
we should rename this file, so that we dont get the following error:

```
imported module 'test_sift_reproducibility' has this __file__ attribute:
  /home/runner/work/gtsfm/gtsfm/tests/repro_tests/frontend/descriptor/test_sift_reproducibility.py
which is not the same as the test file we want to collect:
  /home/runner/work/gtsfm/gtsfm/tests/repro_tests/frontend/detector_descriptor/test_sift_reproducibility.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
```

We already have a test file with this same name for the descriptor-only test.